Makefile.dep

```diff
@@ -589,6 +589,11 @@ ifneq (,$(filter random,$(USEMODULE)))
     ifneq (,$(filter prng_tinymt32,$(USEMODULE)))
         USEMODULE += tinymt32
     endif
+    # sources for random seed
+    FEATURES_REQUIRED += periph_hwrng
```

Comment:
now that I think about it, how about `FEATURES_OPTIONAL` as done with `periph_rtc` in the `default` example?